src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP4200/CreateTransactionEOF1Filler.yml

```
@@ -0,0 +1,419 @@
+---
+CreateTransactionEOF1:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      # Valid EOF deployed code with RJUMP having JUMPDEST as target
+      # - Positive:
+      # Deployed code: RJUMP(3) STOP RETURN STOP JUMPDEST PUSH1(1) PUSH1(1) SSTORE STOP - ef000101000d005c000300f3005b600160015500
+      # Initcode: [Data: contract_code, Code: PUSH1(contract_size) PUSH1(data_offset) PUSH1(0) CODECOPY PUSH1(contract_size) PUSH1(0) RETURN]
+      - ':label eof1_initcode_deploy_eof1_rjump_jumpdest_pos :raw 0xef000101000c020014006014601660003960146000f3ef000101000d005c000300f3005b600160015500'
+      # - Negative:
+      # Deployed code: RJUMP(8) STOP JUMPDEST PUSH1(1) PUSH1(1) SSTORE STOP JUMPDEST RJUMP(-11) STOP - ef0001010010005c0008005b6001600155005b5cFFF500
+      # Initcode: [Data: contract_code, Code: PUSH1(contract_size) PUSH1(data_offset) PUSH1(0) CODECOPY PUSH1(contract_size) PUSH1(0) RETURN]
+      - ':label eof1_initcode_deploy_eof1_rjump_jumpdest_neg :raw 0xef000101000c020017006017601660003960176000f3ef0001010010005c0008005b6001600155005b5cFFF500'
+      # - Zero:
+      # Deployed code: RJUMP(0) JUMPDEST PUSH1(1) PUSH1(1) SSTORE STOP - ef000101000a005c00005b600160015500
+      # Initcode: [Data: contract_code, Code: PUSH1(contract_size) PUSH1(data_offset) PUSH1(0) CODECOPY PUSH1(contract_size) PUSH1(0) RETURN]
+      - ':label eof1_initcode_deploy_eof1_rjump_jumpdest_zer :raw 0xef000101000c020011006011601660003960116000f3ef000101000a005c00005b600160015500'
+
+      # Valid EOF deployed code with RJUMPI having JUMPDEST as target
+      # - Positive:
+      # Deployed code: PUSH1(1) RJUMPI(3) STOP RETURN STOP JUMPDEST PUSH1(1) PUSH1(1) SSTORE STOP - ef000101000f0060015d000300f3005b600160015500
```

To avoid unreachable code
```suggestion
      # Deployed code: PUSH1(1) RJUMPI(3) NOP NOP STOP JUMPDEST PUSH1(1) PUSH1(1) SSTORE STOP - ef000101000f0060015d00035B5B005b600160015500
```